test/service/internal_key_event_handlers/tab_handler_test.dart

```diff
@@ -151,122 +151,122 @@ void main() async {
         BuiltInAttributeKey.bulletedList,
       );
     });
-  });
 
-  testWidgets('press tab in checkbox/todo list', (tester) async {
-    const text = 'Welcome to Appflowy 😁';
-    final editor = tester.editor
-      ..insertTextNode(
-        text,
-        attributes: {
-          BuiltInAttributeKey.subtype: BuiltInAttributeKey.checkbox,
-          BuiltInAttributeKey.checkbox: false,
-        },
-      )
-      ..insertTextNode(
-        text,
-        attributes: {
-          BuiltInAttributeKey.subtype: BuiltInAttributeKey.checkbox,
-          BuiltInAttributeKey.checkbox: false,
-        },
-      )
-      ..insertTextNode(
-        text,
-        attributes: {
-          BuiltInAttributeKey.subtype: BuiltInAttributeKey.checkbox,
-          BuiltInAttributeKey.checkbox: false,
-        },
+    testWidgets('press tab in checkbox/todo list', (tester) async {
+      const text = 'Welcome to Appflowy 😁';
+      final editor = tester.editor
+        ..insertTextNode(
+          text,
+          attributes: {
+            BuiltInAttributeKey.subtype: BuiltInAttributeKey.checkbox,
+            BuiltInAttributeKey.checkbox: false,
+          },
+        )
+        ..insertTextNode(
+          text,
+          attributes: {
+            BuiltInAttributeKey.subtype: BuiltInAttributeKey.checkbox,
+            BuiltInAttributeKey.checkbox: false,
+          },
+        )
+        ..insertTextNode(
+          text,
+          attributes: {
+            BuiltInAttributeKey.subtype: BuiltInAttributeKey.checkbox,
+            BuiltInAttributeKey.checkbox: false,
+          },
+        );
+      await editor.startTesting();
+      var document = editor.document;
+
+      var selection = Selection.single(path: [0], startOffset: 0);
```

Comment:
```suggestion
      Selection selection = Selection.single(path: [0], startOffset: 0);
```
Again, not sure it makes it more readable especially if you're going to change the variable many times.